session/bootstrap.go

```diff
@@ -248,6 +248,19 @@ const (
 		DEFAULT_ROLE_USER char(32) COLLATE utf8_bin NOT NULL DEFAULT '',
 		PRIMARY KEY (HOST,USER,DEFAULT_ROLE_HOST,DEFAULT_ROLE_USER)
 	)`
+
+	// CreateStatsTopNStoreTable stores topn data of a cmsketch with top n.
+	// It stores data of topn of stats_histograms(table_id, is_index, hist_id)
+	// value_id is the position this data lies in the TopN array of serialized CMSketch.
+	// content is the raw data, since a single column can be atmost 6MiB, we do not store other data.
+	CreateStatsTopNStoreTable = `CREATE TABLE if not exists mysql.stats_topnstore (
+		table_id bigint(64) NOT NULL,
+		is_index tinyint(2) NOT NULL,
+		hist_id bigint(64) NOT NULL,
+		value_id bigint(64) NOT NULL,
+		content longblob,
```

Comment:
why not store the count for this content?

Reply:
It is stored with value, see https://github.com/pingcap/tidb/pull/10255/files#diff-ceb17c699db78bf59291b0dcb6f8dda2R305

Reply:
In some corner case, the length of one column can be as long as 6MiB, if we store the count in it, the length of the row will exceed the limit.

Reply:
No, I removed data before produce the final protobuf bytes.
That is, we still need to store the value when exportin g JSON.

Reply:
For now, it also has the risk to exceed the limit, because we stored more than just `content`.

I think you can replace `value_id` to `count`